x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -206,7 +213,7 @@ export class HeadlessChromiumDriverFactory {
     // just log closing of the process
     const processClose$ = Rx.fromEvent<void>(childProcess, 'close').pipe(
       tap(() => {
-        this.logger.debug('child process closed', ['headless-browser-process']);
+        logger.debug('child process closed', ['headless-browser-process']);
```

Comment:
main changes (4) - pass a logger object to `createPage` instead have a `this.logger`